main.lua

```diff
@@ -286,7 +287,7 @@ function LSPClient:initialize(server)
         params.initializationOptions = server.initializationOptions
     end
 
-    client:request("initialize", params)
+    client:request(Request("initialize", params))
```

Comment:
If we give the `onResult` handler for initialize request here, it won't need to be special cased later in `LSPClient:receiveMessage`.